backend/typescript/services/implementations/camperService.ts

```diff
@@ -12,7 +12,23 @@ class CamperService implements ICamperService {
   async createCamper(camper: CreateCamperDTO): Promise<CamperDTO> {
     let newCamper: Camper;
     let existingCamp: Camp | null;
+
     try {
+      existingCamp = await MgCamp.findById(camper.camp);
+
+      if (existingCamp) {
+        const existingCampObj = existingCamp.toObject();
+
+        if (
+          existingCamp &&
+          existingCamp.campers.length === existingCampObj.capacity
```

Comment:
I think we should have 
`existingCamp.campers.length >= existingCampObj.capacity`
to be safe. There could be an edge case where the capacity changes to a lower value or we erroneously add too many campers, and we want to be able to catch that when creating new campers.

Reply:
done!